src/plugins/jingle/chat-header-notification.js

```diff
@@ -0,0 +1,60 @@
+import { CustomElement } from 'shared/components/element.js';
+import { _converse, api, converse } from "@converse/headless/core";
+import tpl_header_button from "./templates/header-button.js";
+import { JINGLE_CALL_STATUS } from "./constants.js";
+
+import './styles/jingle.scss';
+const { Strophe, $msg } = converse.env;
+
+export default class CallNotification extends CustomElement {
+    
+    static get properties() {
+        return {
+            'jid': { type: String },
+        }
+    }
+
+    initialize() {
+        this.model = _converse.chatboxes.get(this.jid);
+        this.listenTo(this.model, 'change:jingle_status', () => this.requestUpdate());
+    }
+    
+    render() {
+        return tpl_header_button(this);
+    }
+
+    endCall() {
+        const jingle_status = this.model.get('jingle_status');
+        if ( jingle_status === JINGLE_CALL_STATUS.OUTGOING_PENDING ) {
+            this.model.save('jingle_status', JINGLE_CALL_STATUS.ENDED);
+            const initiator_stanza = this.model.messages.findWhere({ 'media': 'audio' });
```

Comment:
@jcbrand am getting `initiator_stanza: 1`

Reply:
Your message is quite cryptic, I don't understand. Please provide more info.

Reply:
> You're getting a `1` back from `findWhere`? That doesn't seem likely.

that is correct

Reply:
<img width="1006" alt="Screen Shot 2022-07-18 at 7 58 55 PM" src="https://user-images.githubusercontent.com/48093317/179534185-d70f1dd9-6ce4-477c-a6c8-33c7e434dd50.png">
@jcbrand